modules/prepare-ocp-for-bare-metal-builds.adoc

```diff
@@ -0,0 +1,199 @@
+:_content-type: PROCEDURE
+[id="prepare-ocp-for-bare-metal-builds"]
+= Configuring bare metal builds for {productname-ocp}
+
+Use the following procedure to configure _bare metal builds_ for {productname-ocp}.
+
+[NOTE]
+====
+If you are using the {productname} Operator on {ocp} with a managed `route` component in your `QuayRegistry` CRD, see "{productname-ocp} _builds_ limitations with self-managed _routes_".
+====
+
+.Prerequisites 
+
+* You have an {ocp} cluster provisioned with the {productname} Operator running.
+* You have set the `tls` component to `unmanaged` and uploaded custom SSL/TLS certificates to the {productname} Operator. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/securing_red_hat_quay/index#ssl-tls-quay-overview[SSL and TLS for {productname}].
+* You are logged into {ocp} as a cluster administrator.
+
+.Procedure 
+
+. Enter the following command to create a project where Builds will be run, for example, `bare-metal-builder`:
++
+[source,terminal]
+----
+$ oc new-project bare-metal-builder
+----
+
+. Create a new `ServiceAccount` in the the `bare-metal-builder` namespace by entering the following command:
++
+[source,terminal]
+----
+$ oc create sa -n bare-metal-builder quay-builder
+----
+
+. Enter the following command to grant a user the `edit` role within the `bare-metal-builder` namespace:
++
+[source,terminal]
+----
+$ oc policy add-role-to-user -n bare-metal-builder edit system:serviceaccount:builder:quay-builder
+----
```

Comment:
If the namespace is `bare-metal-builder`, the command should be
```
$ oc policy add-role-to-user -n bare-metal-builder edit system:serviceaccount:bare-metal-builder:quay-builder
```
Conform to format `oc policy add-role-to-user -n ${BUILDER_PROJECT} edit system:serviceaccount:${BUILDER_PROJECT}:${SERVICE_ACCOUNT_NAME}`

Reply:
Thanks! Fixed.